### ISSUE TITLE
new pyflakes causes exceptions on clean run

### ISSUE THREAD

**Issue by arahayrabedian** [pre-commit/pre-commit-hooks#59]
Hey, had just forked and was pulling my hair out wondering what I broke while I was testing a hook I wrote, turns out pyflakes just released [0.9.0](https://github.com/pyflakes/pyflakes/releases/tag/0.9.0), which causes me to get the below when trying to run any of my hooks when running fresh.

```
Traceback (most recent call last):
  File "/home/user/.pre-commit/repoHJDFOT/py_env-default/bin/trailing-whitespace-fixer", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/home/user/.pre-commit/repoHJDFOT/py_env-default/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3074, in <module>
    @_call_aside
  File "/home/user/.pre-commit/repoHJDFOT/py_env-default/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3060, in _call_aside
    f(*args, **kwargs)
  File "/home/user/.pre-commit/repoHJDFOT/py_env-default/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3087, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/home/user/.pre-commit/repoHJDFOT/py_env-default/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 647, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/home/user/.pre-commit/repoHJDFOT/py_env-default/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 660, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/home/user/.pre-commit/repoHJDFOT/py_env-default/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 838, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (pyflakes 0.9.0 (/home/user/.pre-commit/repoHJDFOT/py_env-default/lib/python2.7/site-packages), Requirement.parse('pyflakes<0.9,>=0.8.1'), set(['flake8']))
```

not sure who would need to tackle this so picking on the thing closest first. lemme know if i need to raise this with someone else.

**Comment by arahayrabedian** [comment]
I handled it temporarily by manually installing an older version of pyflakes in the pre-commit env.

**Comment by arahayrabedian** [comment]
my bad, gonna open this on pre-commit, and not hooks repo.

**Comment by asottile** [comment]
Ah no, this is the correct place.  /me shakes fist at pyflakes

**Comment by arahayrabedian** [comment]
oh, whoops, close https://github.com/pre-commit/pre-commit/issues/237 too, since it's actually the same. apologies on that.

**Comment by arahayrabedian** [comment]
that was supposed to be a question

**Comment by arahayrabedian** [comment]
and that wraps it up, thanks guys :)

**Comment by asottile** [comment]
Yeah the bug is in this hook repo itself, this was the right place.

**Comment by asottile** [comment]
I've released v0.4.2 containing this fix for those of us that prefer tags to shas
